lgc/test/TextureRange.lgc

```diff
@@ -1,50 +1,75 @@
 
 ; RUN: lgc %s -print-after=lgc-builder-replayer -o /dev/null 2>&1 - <%s | FileCheck --check-prefixes=CHECK %s
 
-; CHECK: [[descTable:%[0-9]+]] = call i8 addrspace(4)* @lgc.descriptor.table.addr(i32 1
-; CHECK: %{{.*}} = getelementptr i8, i8 addrspace(4)* [[descTable]], i32 32
+; CHECK: [[desc0:%[0-9]+]] = call i8 addrspace(4)* @lgc.descriptor.table.addr(i32 6
+; CHECK: %{{.*}} = getelementptr i8, i8 addrspace(4)* [[desc0]], i32 16
+; CHECK: call <2 x i32> @lgc.root.descriptor.v2i32(i32 6)
+; CHECK: [[desc1:%[0-9]+]] = call i8 addrspace(4)* @lgc.descriptor.table.addr(i32 1
+; CHECK: %{{.*}} = getelementptr i8, i8 addrspace(4)* [[desc1]], i32 32
 
 ; RUN: lgc -mcpu=gfx1030 -o - - <%s | FileCheck --check-prefixes=SHADER_TEST %s
 ; SHADER_TEST: s_load_dwordx8 [[desc:.*]], {{.*}}, 0x20
 ; SHADER_TEST: image_sample {{.*}}, {{.*}}, [[desc]], {{.*}} dmask:0xf
-
+; SHADER_TEST: s_load_dwordx4 {{.*}}, {{.*}}, 0x10
+; SHADER_TEST: .registers:
+; SHADER_TEST: (SPI_SHADER_USER_DATA_PS_4): 0x6
+; SHADER_TEST: (SPI_SHADER_USER_DATA_PS_5): 0x7
```

Comment:
I don't see where s4 and s5 are used in the GCN.  I cannot be sure they are being used correctly.